pkg/ttnpb/go.mod

```diff
@@ -0,0 +1,46 @@
+module go.thethings.network/lorawan-stack/v3/pkg/ttnpb
+
+go 1.14
+
+replace go.thethings.network/lorawan-stack/v3 => ../../
```

Comment:
This seems to be the only replacement we need here.
Not sure if this works as expected when imported by a third party though. I guess it should.